webplugin/js/app/kommunicate.js

```diff
@@ -514,5 +514,28 @@ $applozic.extend(true,Kommunicate,{
     displayKommunicateWidget: function(display) {
         var kommunicateIframe = parent.document.getElementById('kommunicate-widget-iframe');
         display ? kommunicateIframe.classList.remove("kommunicate-hide-custom-iframe") : kommunicateIframe.classList.add("kommunicate-hide-custom-iframe");
+    },
+    // check if the message needs to be processed by addMessage
+    visibleMessage: function(msg){
+        if(!msg) return false;
+        if(msg.metadata && msg.metadata.feedback){
+            return false;
+        }
+        if (!msg.message && msg.metadata.hasOwnProperty("KM_ASSIGN_TO")) { // KM_ASSIGN_TO parameter comes when we change assignee by bot message.
+            return false;
+        }
+        if (msg.type === 6 || msg.type === 7) {
+            return false;
+        }
+        if ((msg.metadata && msg.metadata.category === 'HIDDEN') || msg.contentType === 102) {
+            return false;
+        }
+        if(msg.metadata && (msg.metadata.KM_ASSIGN || msg.metadata.KM_STATUS)){
+            return false;
+        }
+        if (msg.contentType === 10 && (msg.metadata && msg.metadata.hide === 'true')) {
```

Comment:
@shubhamtibra Just a small code improvement.
Can you create constants for message type and contentType?
Check if it's already present, otherwise, we can create constants in kmUtils.js file.

Reply:
done